test/integration_test.py

```diff
@@ -1104,20 +1104,25 @@ def test_can_bundle_configured_catalogs(self):
 
     @mock.patch.dict(os.environ,
                      azul_canned_sources=f'{canned_repo}:')
-    @mock.patch.object(azul.Config, 'catalogs', new_callable=PropertyMock)
-    # Must be mocked independently due to caching
-    @mock.patch.object(azul.Config, 'default_catalog', new_callable=PropertyMock)
-    def test_can_bundle_canned_repository(self, mock_default_catalog, mock_catalogs):
+    def test_can_bundle_canned_repository(self):
         mock_catalog_name = 'testcanned'
         mock_catalog = config.Catalog(name=mock_catalog_name,
                                       atlas='hca',
                                       plugins={
                                           'metadata': 'hca',
                                           'repository': 'canned'
                                       })
-        mock_default_catalog.return_value = mock_catalog_name
-        mock_catalogs.return_value = {mock_catalog_name: mock_catalog}
-        self._test_catalog(mock_catalog)
+
+        with mock.patch.object(azul.Config,
+                               'catalogs',
+                               new=PropertyMock(return_value={
+                                   mock_catalog_name: mock_catalog
```

Comment:
We should be able to inline `mock_catalog_name` and replace with `mock_catalog.name`.